include/rtio.h

```diff
@@ -0,0 +1,479 @@
+/*
+ * Copyright (c) 2019  Thomas Burdick <thomas.burdick@gmail.com>
```

Comment:
I believe this file should be moved to include/drivers/ directory.

Reply:
I believe your right for master, this is going into a topic branch that likely needs rebasing again to match master

Reply:
I did move this for now. It can be moved back to include/ rather than include/drivers/ if it makes more sense